saleor/graphql/query_cost_map.py

```diff
@@ -256,6 +256,7 @@
         "productType": {"complexity": 1},
         "thumbnail": {"complexity": 1},
         "variants": {"complexity": 1},
+        "productVariants": {"complexity": 100},
```

Comment:
`"complexity": 100` should be set for fields without pagination fetching multiple objects. When we have pagination we should use `multipliers`. 
```suggestion
        "productVariants": {"complexity": 1, "multipliers": ["first", "last"]},
```